core/src/main/java/com/linecorp/armeria/server/docs/DocService.java

```diff
@@ -70,10 +73,20 @@
  */
 public class DocService extends AbstractCompositeService<HttpRequest, HttpResponse> {
 
-    private static final ObjectMapper mapper = new ObjectMapper();
+    private static final int SPECIFICATION_INDEX = 0;
```

Comment:
`1` was for the OpenAPI JSON and `2` was for the OpenAPI YAML, I just didn't revert it because I have to change it again when I introduce the OpenAPI. Do you want me to make it back or can I just leave it as it is?